README.md

```diff
@@ -22,6 +22,23 @@ npm i
 npm run install-extension
 ```
 
+## Test your changes on your local install of VS Code
+
+Firstly, uninstall the official release of the extension and reload VS Code if you have it installed. You will need to do this step each time you want to re-install and test your local version.
+
+Install `vsce`:
+
+```sh
+npm install -g vsce
+```
+
+`checkout` your branch, if not already on it and run the following in the root of the project:
+
+```sh
+vsce package  # Generates a .vsix file
+code --install-extension sway-vscode-extension-{version}.vsix
```

Comment:
It's not immediately clear to me, but is `{version}` something the user has to change manually?
```suggestion
code --install-extension sway-vscode-extension-{version}.vsix
```

Reply:
That's correct, I've changed it to be `<version>`, is there a clearer way?

Reply:
It's a bit clearer; a shell variable like 

```sh
VERSION=<version>
code --install-extension sway-vscode-extension-$VERSION.vsix
```

would be even clearer. The only issue is that there's no documentation on how the user would decide on the version.

Reply:
The version is included in the file name when you run `vsce package`

Reply:
I'll add it as a comment

Reply:
I only added it as a variable due to the fact that it could change, but it's automatically generated when you run the command